charts/alfresco-adf-app/templates/deployment.yaml

```diff
@@ -42,12 +42,20 @@ spec:
               protocol: TCP
           livenessProbe:
             httpGet:
-              path: /
+              path: {{ .Values.livenessProbe.path }}
               port: {{ .Values.service.targetPort }}
+            initialDelaySeconds: {{ .Values.livenessProbe.initialDelaySeconds }}
+            periodSeconds: {{ .Values.livenessProbe.periodSeconds }}
+            failureThreshold: {{ .Values.livenessProbe.failureThreshold }}
+            timeoutSeconds: {{ .Values.livenessProbe.timeoutSeconds }}
```

Comment:
why not using the toYaml syntax also here? you can use the named port as well

Reply:
Using the named port here breaks the probes, it has to connect to port 8080